addons/crm/models/crm_lead.py

```diff
@@ -452,7 +402,7 @@ def _update_probability(self):
             if lead.id in lead_probabilities:
                 lead_proba = lead_probabilities[lead.id]
                 proba_vals = {'automated_probability': lead_proba}
-                if tools.float_compare(lead.probability, lead.automated_probability, 2) == 0:
+                if tools.float_compare(lead.probability, lead.automated_probability, 2) == 0 and lead.active:
```

Comment:
same here: why not using is_automated_probability ?

Reply:
Why ? Why not ? Am I a dog ?

I am not a dog. I will use is_automated_probability .

Or am I a dog ?

Not sure...

Coin ...

Not a dog. Definitively.